package.json

```
@@ -28,6 +28,8 @@
     "dev": "PLUGIN=odf I8N_NS=plugin__odf-console yarn server:plugin",
     "dev-mco": "PLUGIN=mco I8N_NS=plugin__odf-multicluster-console yarn server:plugin",
     "http-server": "./http-server.sh ./dist",
+    "serve-local-build": "http-server plugins/odf/dist -p 9001 -c-1 --cors",
+    "serve-local-build-mco": "http-server plugins/mco/dist -p 9001 -c-1 --cors",
```

```suggestion
    "serve-local-build": "./http-server plugins/odf/dist -p 9001 -c-1 --cors",
    "serve-local-build-mco": "./http-server plugins/mco/dist -p 9001 -c-1 --cors",
```
We might have to change the script to remove the requirement for certs.

---

can't we have script accept certs (as it is doing now)...
and here directly execute `http-server plugins/odf/dist -p 9001 -c-1 --cors` instead of calling the `http-server.sh` script... like what Alfonso already did ??
we don't have to update script that way

---

Updated http-server.sh and adapted Dockerfiles.